_pages/tycrawford.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: tycrawford
+pageUrl: tycrawford.github.io
+timestamp: 2018-04-12
+---
```

Comment:
You should have another line at the end of the file or your build won't success. 🏅